### PR TITLE
New Game:[The Mystery Adventure Game]

### LINKED ISSUE #5055: [New game]: The Enigma of Blackwood Manor
### 🎮 Game Request

Game logic and basic description
Welcome to "The Enigma of Blackwood Manor," an immersive mystery adventure game set in the eerie and enigmatic Blackwood Manor. As the protagonist, you are a renowned detective called to solve the mysterious disappearance of the manor's owner, Lady Blackwood. With a rich storyline, intriguing characters, and challenging puzzles, players will navigate through the manor's dark corridors, uncover secrets, and piece together clues to solve the mystery.

### Point down the features

Game points

1. **Compelling Storyline**: Unravel the mystery of Lady Blackwood's disappearance through an engaging narrative.
  
2. **Exploration**: Navigate through different rooms and areas of the manor by clicking on doorways and interactive objects.

3. **Clue Collection**: Discover and collect clues and items hidden throughout the manor to progress in the game.

4. **Puzzle Solving**: Solve various puzzles and mini-games, such as lock picking, code breaking, and pattern matching.

5. **Dialogue System**: Interact with characters and ghosts, engaging in branching dialogues to gather information and influence the story.

6. **Inventory Management**: Store and use collected items to interact with other objects and solve puzzles.

7. **Objectives and Clues Tracking**: Keep track of current objectives and collected clues in the HUD for easy reference.

8. **Atmospheric Visuals and Sounds**: Enhance the eerie setting with background images, character sprites, animations, sound effects, and background music.

9. **Multiple Endings**: Offer different story outcomes based on player choices and actions throughout the game.

10. **User-Friendly Interface**: Intuitive navigation, clear HUD elements, and responsive design for a seamless gameplay experience.

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by priyashuu**
## PR Description 📜
The Mystery Adventure Game invites players to explore immersive environments with rich, full-screen backgrounds, interact with objects to collect items, and solve puzzles to uncover secrets and advance the story. Navigate between rooms using simple controls, and manage gameplay with start, stop, and restart options. Built with HTML, CSS, and JavaScript, this game offers a seamless and engaging experience for mystery enthusiasts.

Fixes #5055

<hr>
 
## Mark the task you have completed ✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in GameZone's README.md
- [x] I have added README.md in my Game folder 
- [x] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸

The Mystery Adventure Game

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @priyashuu ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

File: README.md [hunk 1]
```diff
@@ -1713,6 +1713,7 @@ This repository also provides one such platforms where contributers come over an
 
 |[Tower_Building_Game](https://github.com/kunjgit/GameZone/tree/main/Games/Tower_Building_Game)|
 |[Cross_Road_Game](https://github.com/kunjgit/GameZone/tree/main/Games/Cross_Road_Game)|
+|[The_Mystery_Adventure_game](https://github.com/kunjgit/GameZone/tree/main/Games/The%20Mystery%20Adventure%20Game)|
```
Comment (ishita-43) [inline README.md:1716]: Change the end with your game folder nae instead as other did  only folder name will change and rest will be the same

**Review by ishita-43 — CHANGES_REQUESTED** [review]
Please make following update

File: README.md (hunk 1 above)
Comment (priyashuu) [reply README.md]: @ishita-43 

Screenshot 2024-08-09 212433
is it okay mam ?

**Review by ishita-43 — APPROVED** [review]
Ready to merge

**Comment by kunjgit** [comment]
Thank you  @priyashuu , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**